examples/aspnetcore/AspNetCore.CSharp/appsettings.json

```diff
@@ -0,0 +1,10 @@
+﻿{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Warning"

```

Comment:
Shouldn't this be lower? To let Logary handle the levels?

Reply:
there are two ways, let aspnet core pass all message to logary or  vice versa, here is aspnet core logging way to handler rules and levels.